Resources/Prototypes/Entities/Structures/Dispensers/chem.yml

```diff
@@ -12,9 +12,6 @@
     emagPack: ChemDispenserEmaggedInventory
   - type: ApcPowerReceiver
   - type: ExtensionCableReceiver
-  - type: Construction
-    graph: Machine
-    node: machine
```

Comment:
removed with nothing else added?

Reply:
The dispenser already inherits from ReagentDispenserBase, which now newly inherits from ConstructibleMachine which adds this & some other missing construction related stuff.